changelog/0087-upgrade-to-es-6.yml

```
@@ -80,8 +86,8 @@ update_steps: |
   5. Run update config and restart the services to apply the changes.
 
       ```sh
-      cchq <env> django-manage update_config
-      cchq <env> django-manage restart_services
+      cchq <env> update_config
```

I think this still has a typo.
```suggestion
      cchq <env> update-config
```
There are a couple of other similar lines below.

---

Thanks @millerdev for catching it. I don't know what I was thinking while adding them 😓 

Made a fix in - https://github.com/dimagi/commcare-cloud/pull/6580